code/modules/surgery/organs/external/_external_organs.dm

```diff
@@ -101,6 +106,8 @@
 		organ_owner.external_organs.Remove(src)
 		organ_owner.update_body_parts()
 
+	ownerlimb = null
```

Comment:
remove_from_limb() should be doing this

Reply:
I need to update the whole drop_organs proc cause that shit broke everything